doc/api/util.md

```diff
@@ -393,8 +393,9 @@ changes:
   * `colors` {boolean} If `true`, the output will be styled with ANSI color
     codes. Colors are customizable, see [Customizing `util.inspect` colors][].
     **Default:** `false`.
-  * `customInspect` {boolean} If `false`, then custom `inspect(depth, opts)`
-    functions will not be called. **Default:** `true`.
+  * `customInspect` {boolean} If `false`, then
+    `[util.inspect.custom](depth, opts)` functions will not be called.
```

Comment:
Can many functions be called during inspections? Does this mean nested objects with own custom functions?

Reply:
You are right, it is the same function (if it does not replace itself during custom inspection ;-) ).

Reply:
Looking at this again: functions do seem to be correct as it is not about a specific custom inspect function but about custom inspect functions in general.